Gemfile

```
@@ -17,5 +17,6 @@ end
 group :type_check do
   if RUBY_VERSION >= "2.6" && %w[ ruby truffleruby ].include?(RUBY_ENGINE)
     gem 'rbs', '~> 3.0'
+    gem 'steep', '~> 1.6'
```

Steep 1.7.1 released; it fixex №3.
```suggestion
    gem 'steep', '~> 1.7', '>= 1.7.1'
```